custom-wasm-transformers/samples/wasm-gen-svc-acc/gen-svc-acc/go.mod

```diff
@@ -0,0 +1,5 @@
+module foo.com/b
```

Comment:
Can we have something other than "foo" (say mymodule.com)

Reply:
done